src/bots/modchat/constants.js

```diff
@@ -0,0 +1,18 @@
+import { dedent } from '../../utils';
+
+/**
+ * Contructs string object with options
+ * @param  {...any} options - data for strings and other options
+ * @return {Object.<string, string>} - string object with all AMA bot strings
+ */
+export const getStrings = (...options) => {
+  options = options || [{}];
```

Comment:
you can just give options a default value of `[]` when it's an argument if you don't use the spread operator. not sure why it's `[{}]`

Reply:
c/p from other bots